src/components/Value.js

```diff
@@ -9,10 +9,11 @@ const Value = ({ initial, onChange, ...props }) => (
     initial={{ value: initial }}
     onChange={onChangeProp(onChange, 'value')}
   >
-    {({ state, setState }) =>
+    {({ state, setState, resetState }) =>
       renderProps(props, {
         value: state.value,
         set: value => setState(s => ({ value: set(value, s.value) })),
+        reset: () => resetState(),
```

Comment:
Why not `reset: resetState`?

Reply:
By analogy, State has setState with callback, so resetState has too, but all other components has no callback, so here we remove callback too